lib/vision/index.js

```diff
@@ -0,0 +1,1376 @@
+        right: findLandmark('CHIN_RIGHT_GONION')
+      },
+      ears: {
+        left: findLandmark('LEFT_EAR_TRAGION'),
+        right: findLandmark('RIGHT_EAR_TRAGION'),
+      },
+      eyebrows: {
+        left: {
+          left: findLandmark('LEFT_OF_LEFT_EYEBROW'),
+          right: findLandmark('RIGHT_OF_LEFT_EYEBROW'),
+          top: findLandmark('LEFT_EYEBROW_UPPER_MIDPOINT')
+        },
+        right: {
+          left: findLandmark('LEFT_OF_RIGHT_EYEBROW'),
+          right: findLandmark('RIGHT_OF_RIGHT_EYEBROW'),
+          top: findLandmark('RIGHT_EYEBROW_UPPER_MIDPOINT')
+        }
+      },
+      eyes: {
+        left: {
+          bottom: findLandmark('LEFT_EYE_BOTTOM_BOUNDARY'),
+          center: findLandmark('LEFT_EYE'),
+          left: findLandmark('LEFT_EYE_LEFT_CORNER'),
+          pupil: findLandmark('LEFT_EYE_PUPIL'),
+          right: findLandmark('LEFT_EYE_RIGHT_CORNER'),
+          top: findLandmark('LEFT_EYE_TOP_BOUNDARY')
+        },
+        right: {
+          bottom: findLandmark('RIGHT_EYE_BOTTOM_BOUNDARY'),
+          center: findLandmark('RIGHT_EYE'),
+          left: findLandmark('RIGHT_EYE_LEFT_CORNER'),
+          pupil: findLandmark('RIGHT_EYE_PUPIL'),
+          right: findLandmark('RIGHT_EYE_RIGHT_CORNER'),
+          top: findLandmark('RIGHT_EYE_TOP_BOUNDARY')
+        }
+      },
+      forehead: findLandmark('FOREHEAD_GLABELLA'),
+      lips: {
+        bottom: findLandmark('LOWER_LIP'),
+        top: findLandmark('UPPER_LIP')
+      },
+      mouth: {
+        center: findLandmark('MOUTH_CENTER'),
+        left: findLandmark('MOUTH_LEFT'),
+        right: findLandmark('MOUTH_RIGHT')
+      },
+      nose: {
+        bottom: {
+          center: findLandmark('NOSE_BOTTOM_CENTER'),
+          left: findLandmark('NOSE_BOTTOM_LEFT'),
+          right: findLandmark('NOSE_BOTTOM_RIGHT')
+        },
+        tip: findLandmark('NOSE_TIP'),
+        top: findLandmark('MIDPOINT_BETWEEN_EYES')
+      }
+    },
+
+    confidence: faceAnnotation.detectionConfidence * 100
+  };
+
+  extend(formattedFaceAnnotation, Vision.convertToBoolean_(LIKELY, {
+    blurry: faceAnnotation.blurredLikelihood,
+    dark: faceAnnotation.underExposedLikelihood,
+    happy: faceAnnotation.joyLikelihood,
+    hat: faceAnnotation.headwearLikelihood,
+    mad: faceAnnotation.angerLikelihood,
+    sad: faceAnnotation.sorrowLikelihood,
+    surprised: faceAnnotation.surpriseLikelihood
+  }));
+
+  return formattedFaceAnnotation;
+};
+
+/**
+ * Format a raw image properties annotation response from the API.
+ *
+ * @private
+ */
+Vision.formatImagePropertiesAnnotation_ = function(imageAnnotation, options) {
+  var formattedImageAnnotation = {
+    colors: imageAnnotation.dominantColors.colors
+      .map(function(colorObject) {
+        var red = colorObject.color.red;
+        var green = colorObject.color.green;
+        var blue = colorObject.color.blue;
+
+        var hex = colorFormat.rgbToHex(red, green, blue);
```

Comment:
If we're only using this for rgb -> hex, have you thought about using [rgb-hex](https://github.com/sindresorhus/rgb-hex) instead?